hack/tools/release/notes.go

```diff
@@ -73,6 +73,8 @@ var (
 
 	prefixAreaLabel = flag.Bool("prefix-area-label", true, "If enabled, will prefix the area label.")
 
+	preReleaseVersion           = flag.Bool("pre-release-version", false, "If enabled, will add a pre-release warning header. (default false)")
+	deprecation                 = flag.Bool("deprecation", true, "If enabled, will add a templated deprecation warning header. (default true)")
```

Comment:
Not sure how often we will use this. Is the plan to use it for other things apart from getting rid of v1alpha3/v1alpha4? Maybe the default should be false?

Reply:
I think this was a UX preference discussed here: https://github.com/kubernetes-sigs/cluster-api/pull/9247#discussion_r1300216670 where it's probably easier to delete this section when it's not needed vs remembering to pass the flag - I'm open to either default though

Reply:
Alright. Let's keep it as is